File: Javascript/Queue/PrintManager.js
class PrintManager {
    constructor(){
        this.queue = []
    }

    queuePrintJob (document) {
        this.queue.push(document)
    }

    run() {
        console.log("Length of Queue", this.queue.length);
        while(this.queue.length != 0) {
            console.log(this.queue.splice(0).toString());
        }
    }
    

}


let document1 = "Document 1"
let document2 = "Document 2"
let document3 = "Document 3"

let printManager = new PrintManager()

printManager.queuePrintJob(document1)
printManager.queuePrintJob(document2)
printManager.queuePrintJob(document3)

printManager.run()

File: Python/Queue/PrintManager.py
class PrintManager:
    def __init__(self):
        self.queue = []


    def queue_print_job(self, document):
        self.queue.append(document)
    
    def run(self):
        if len(self.queue) == 0:
            print("The Queue is Empty")
        while (len(self.queue) != 0):
            print(self.queue.pop(0))
        
        

printManager = PrintManager()

ducument1 = "First Document"
ducument2 = "Second Document"
ducument3 = "Third Document"

printManager.queue_print_job(ducument1)
printManager.queue_print_job(ducument2)
printManager.queue_print_job(ducument3)

printManager.run()